xarray/core/dataset.py

```diff
@@ -2497,6 +2497,62 @@ def isel(
             in this dataset, unless vectorized indexing was triggered by using
             an array indexer, in which case the data will be a copy.
 
+        Examples
+        --------
+
+        >>> dataset = xr.Dataset(
+        ...     {
+        ...         "math_scores": (
+        ...             ["student", "test"],
+        ...             [[90, 85, 92], [78, 80, 85], [95, 92, 98]],
+        ...         ),
+        ...         "english_scores": (
+        ...             ["student", "test"],
+        ...             [[88, 90, 92], [75, 82, 79], [93, 96, 91]],
+        ...         ),
+        ...     },
+        ...     coords={
+        ...         "student": ["Alice", "Bob", "Charlie"],
+        ...         "test": ["Test 1", "Test 2", "Test 3"],
+        ...     },
+        ... )
+        # A specific element from the dataset is selected
```

Comment:
If you notice in the [doctests CI run](https://github.com/pydata/xarray/actions/runs/5440530588/jobs/9893551343?pr=7881#step:7:68) there appears to be a failure here, saying that the code above expected no result, but instead it found "# A specific element from the dataset is selected".

Obviously we don't want the doctest to think that our comment is the expected result of running the code - I think the solution might just be to add a newline here.

```suggestion
        ... )
        
        # A specific element from the dataset is selected
```